core/domain/rating_services_test.py

```diff
@@ -150,52 +158,67 @@ def test_rating_assignations_do_not_conflict(self):
             rating_services.get_overall_ratings_for_exploration(exp_id_b),
             {'1': 0, '2': 0, '3': 1, '4': 0, '5': 1})
 
-    def test_invalid_ratings_are_forbidden(self):
-        with self.assertRaisesRegex(
+    def test_invalid_ratings_are_forbidden(self) -> None:
+        with self.assertRaisesRegex(  # type: ignore[no-untyped-call]
             ValueError, 'Expected a rating 1-5, received 0'
             ):
             rating_services.assign_rating_to_exploration(
                 self.USER_ID_1, self.EXP_ID, 0)
 
-        with self.assertRaisesRegex(
+        with self.assertRaisesRegex(  # type: ignore[no-untyped-call]
             ValueError, 'Expected a rating 1-5, received 7'
             ):
             rating_services.assign_rating_to_exploration(
                 self.USER_ID_1, self.EXP_ID, 7)
 
-        with self.assertRaisesRegex(
+        with self.assertRaisesRegex(  # type: ignore[no-untyped-call]
             ValueError, 'Expected the rating to be an integer, received 2'
             ):
+            # The argument new_rating of assign_rating_to_exploration() can
+            # only accept integer values, but for testing purposes here we are
+            # providing string value. Thus to silent MyPy error, we added an
+            # ignore statement here.
```

Comment:
Will this be removed after the codebase is fully typed?

Reply:
Yeah, sure. TODO is added at instance check!